# Test 3.21.17.24

### objective
Control sample with no interference and no api.

mydt = Digital_Twin(
    name= "5s_determ",
    template= True, 
    Freq_Sync= 30, 
    Freq_Valid= 60, 
    Freq_Service= 10000, 
    delta_t_treshold=21,
    ip_address=factory_ip,
    flag_API= True,
    rct_threshold= 0,
    rct_queue= 2,
    flag_external_service= False)

### observation
- ran for 10 minutes.
- no error in real database
- error in template used because queue capacity for machine 1 was not changed.
- Hence, digital twin also gave out error.